packages/runner-ct/src/SpecList/SpecList.tsx

```diff
@@ -11,16 +12,41 @@ interface SpecsListProps {
   selectedSpecs: string[]
   specs: Cypress.Cypress['spec'][]
   onSelectSpec: OnSelectSpec
-  disableTextSelection: boolean
+  disableTextSelection?: boolean
 }
 
-export const SpecList: React.FC<SpecsListProps> = (props) => {
+export const SpecList: React.FC<SpecsListProps> = observer((props) => {
```

Comment:
what does wrapping this with `observer` do? does it fix something specifically or just best practice?

Reply:
Yes, I was facing a problem of just unexpected crash inside useEffect that uses observable. It was just a crash that handled by internal react `try {} catch` without any console.log. If you interested – try to delete it and observe

That's why I asked you to add this observer 😉